lom/Makefile

```diff
@@ -0,0 +1,53 @@
+GO := /usr/local/go/bin/go
+INSTALL := /usr/bin/install
+
```

Comment:
Running make will pull dependencies from go.mod ? Since we have few dependencies on external packages.

Reply:
You can test it out. 
BTW, the Makefile does not do install today.

Reply:
If code uses external packages, you mean make build will pull them?

Reply:
YES. It can't build standalone binary w/o.
Again I am not a SME - learning as we go